Mathlib/Data/Finsupp/Lex.lean

```diff
@@ -104,6 +104,25 @@ theorem lt_of_forall_lt_of_lt (a b : Lex (α →₀ N)) (i : α) :
     (∀ j < i, ofLex a j = ofLex b j) → ofLex a i < ofLex b i → a < b :=
   fun h1 h2 ↦ ⟨i, h1, h2⟩
 
+theorem lex_lt_iff {a b : Lex (α →₀ N)} :
```

Comment:
Can this be placed any earlier? It seems like a very basic lemma, and I don't think it should require the `PartialOrder` assumption.

Reply:
It's just below `lex_le_iff`…

Reply:
Something more general holds for the general `Lex r s` type but is it really worthy to introduce that generality?